website/docs/docs/build/python-models.md

```diff
@@ -396,6 +418,34 @@ def model(dbt, session):
 
 </TabItem>
 
+<TabItem value="BigQuery Dataframe">
+
+<File name='models/my_python_model.py'>
+
+```python
```

Comment:
Please update this to align with the other examples so it's easier for users to see the differences between dialects (if there are any). This means using the holidays package example